### PR TITLE
Update helpdata.txt

### LINKED ISSUE #1256: Incomprehensible string in helpdata.txt
I don't understand the first sentence in this paragraph:
https://github.com/longturn/freeciv21/blob/8854672a801aad805648a8fe45d2c308fccecb39/data/helpdata.txt#L155
The English looks fine, but I have no idea what it's trying to say.

### PR DESCRIPTION
**Pull request by jwrober**
Review of the entire `helpdata.txt` file to bring it in line with the documentation style guide and language used in the Client Manual. Also closes #1256

### CODE REVIEW

File: data/helpdata.txt [hunk 1]
```diff
@@ -1157,30 +1158,32 @@ text = _("\
...
 "), _("\
- - You will sometimes discover enemy technology when you capture a city\n\
- - Your agents can steal advances\n\
- - Wonders can provide you with technology\n\
- - Another player might grant technology in the terms of a pact.\
+ - You will sometimes discover enemy technology when you capture a city.\n\
+ - Your agents can steal advances.\n\
+ - Wonders can provide you with technology.\n\
+ - Another player might grant technology in the terms of a pact.\n\
+ - You may find scrolls of wisdom in a hut.\
```
Comment (daavko) [inline data/helpdata.txt:1167]: Consider pluralizing to "You may find scrolls of wisdom in huts."

File: data/helpdata.txt [hunk 2]
```diff
@@ -1157,30 +1158,32 @@ text = _("\
...
 "), _("\
 Otherwise advances must be discovered through the efforts of your own people. Many of the mentioned methods \
 of technology transfer are also ruleset defined, meaning that they may be disabled.\
 "), _("\
-Most technology progress comes from trade (see the Trade section) and, in the classic ruleset, from scientist \
-specialists (see the Specialists section). While it is possible to change which advance you are currently \
-researching, by default all progress is lost by doing so.\
+Most technological progress comes from trade (see the Trade section) and, in the classic ruleset, from \
+scientist specialists (see the Specialists section). While it is possible to change which advance you are \
```
Comment (daavko) [inline data/helpdata.txt:1173]: Typo: Scientist should be capitalized

File: data/helpdata.txt [hunk 3]
```diff
@@ -1240,13 +1243,13 @@ by default). If the server is started with the -m flag, it will report to the me
 $DefaultMetaserver\
 ", _("\
 The client can fetch this page, too: use the Metaserver button in the connection dialog. If it never shows \
-any results, check whether your WWW browser is using a HTTP proxy; to make the client use the same proxy, \
+any results, check whether your web browser is using a HTTP proxy; to make the client use the same proxy, \
```
Comment (daavko) [inline data/helpdata.txt:1246]: Consider breaking up the sentence on the semicolon

File: data/helpdata.txt [hunk 4]
```diff
@@ -1517,7 +1501,7 @@ using boldface, italic, colors, links, etc. Such changes are performed using esc
     Abbreviation sequence: '[l tgt=\"city\"] ... [/l]'\n\
     Short cut: Control-Alt-Right-click on a city on the map.\n\
     The 'id' parameter must be set to the id of the city you are pointing.\n\
-    An optional 'name' parameter can be set to bind the city name in the case the destination users don't \
+    An optional 'name' parameter can be set to bind the city name in the case the destination users do not \
```
Comment (daavko) [inline data/helpdata.txt:1504]: Typo: redundant "the" in "in the case"

File: data/helpdata.txt [hunk 5]
```diff
@@ -1539,7 +1523,7 @@ to the tile (17, 3).\
     Abbreviation sequence: '[l tgt=\"unit\"] ... [/l]'\n\
     Short cut: Shift-Control-Alt-Right-click on a unit on the map.\n\
     The 'id' parameter must be set to the id of the unit you are pointing to. An optional 'name' parameter \
-can be set to bind the unit name in the case the destination users don't know this unit on their map.\n\
+can be set to bind the unit name in the case the destination users do not know this unit on their map.\n\
```
Comment (daavko) [inline data/helpdata.txt:1526]: Typo: redundant "the" in "in the case"

File: data/helpdata.txt [hunk 6]
```diff
@@ -1553,18 +1537,18 @@ Of course, the different escape sequences can be combined in the same sentence,
 [help_worklist_editor]
 name = _("Worklist Editor")
 text = _("\
-The worklist editor is used to edit both worklists for each city (from the city dialog) and global worklists. \
+The worklist editor is used to edit both worklists for each city (from the City Dialog) and global worklists. \
```
Comment (daavko) [inline data/helpdata.txt:1540]: I'd leave out "both" in this sentence, it reads a little weird with it

File: data/helpdata.txt [hunk 7]
```diff
@@ -1553,18 +1537,18 @@ Of course, the different escape sequences can be combined in the same sentence,
...
 Using this editor you can create lists specifying what to build in the turns to come.\
 "), _("\
-To add an item to the worklist, click the plus ( + ) icon on the Production tab in the city dialog. A list of \
-available items will appear. You can hover over them for some brief help and then click the desired item  to \
+To add an item to the worklist, click the plus ( + ) icon on the Production tab in the City Dialog. A list of \
+available items will appear. You can hover over them for some brief help and then click the desired item to \
 add to the worklist of projects to build.\
 "), _("\
-To remove an item from the worklist simply double-click the item to remove. Use the buttons below the \
+To remove an item from the worklist simply double-click the item to remove it. Use the buttons below the \
```
Comment (daavko) [inline data/helpdata.txt:1547]: Consider rephrasing to "You can remove an item from the worklist by double-clicking on it."

**Review by daavko — COMMENTED** [review]
Too many occurences to list individually: game-specific terms should be capitalized, as discussed on Discord.

File: data/helpdata.txt [hunk 8]
```diff
@@ -131,34 +131,34 @@ improvements on some tiles may require technology.\
...
-is currently producing +4 food surplus each turn. This means that at turn change you will lose 2 food as waste \
-at turn change. This is a good opportunity to move the citizens around to get food to +2 surplus. This could \
-be accomplished by taking a citizen off a tile producing 2 food and turn him into a tax man for a turn to get \
+If you click on the central tile of the land (the \"City Center\"), the citizens will be rearranged to \
+maximize food production. You will also want to look at the level of food in the granary and the amount of \
+surplus food the city is producing each turn. The city will lose excess food at turn change. For example:
+The granary in the city has 18/20 food (needs two food to grow to next city size) and the city is currently \
+producing +4 food surplus each turn. This means that at turn change you will lose 2 food as waste at turn \
+change. This is a good opportunity to move the citizens around to get food to +2 surplus. This could \
+be accomplished by taking a citizen off a tile producing 2 food and turn it into a taxman for a turn to get \
```
[the remark this answers is not in the document]
Comment (jwrober) [reply data/helpdata.txt:161]: "they" is plural - e.g. more than one. "it" refers to a single (non gendered) entity in my view.

[the remark this answers is not in the document]
Comment (daavko) [reply data/helpdata.txt:161]: Not necessarily, "they" can also be singular, when gender is unknown. But I'll leave it up to you, I don't feel too strongly about this one

**Pull request merged**